src/fallbacks.jl

```diff
@@ -111,6 +111,16 @@ coordinates(t::AbstractFeatureCollectionTrait, fc) = [coordinates(f) for f in ge
 
 Base.convert(T::Type, ::AbstractGeometryTrait, geom) = error("Conversion is enabled for type $T, but not implemented. Please report this issue to the package maintainer.")
 
+# Package level `GeoInterface.convert` method
+# Packages must implement their own `traittype` method
+# that accepts a GeoInterface.jl trait and returns the
+# corresponding geometry type
+convert(package::Module, geom) = convert(Val{nameof(package)}(), geom)
+function convert(package::Val, geom)
+    t = trait(geom)
+    convert(package.traittype(t), t, geom)
+end
```

Comment:
I think there is something wrong here. I would have expected 
```julia
function convert(package::Module, geom)
    t = trait(geom)
    convert(package.traittype(t), t, geom)
end
```
Not sure if we also want `convert(::Val...` methods? These would need an `eval` somewhere to get the module.

Reply:
Or we could recommend that a package `Package` implements `GI.traittype(::Val{:Package}, trait)`. This way we could have a fully type stable
```julia
function convert(pkgval::Val, geom)
    T = traittype(pkgval, trait(geom))
    convert(T, geom)
end
```
and a non type stable
```julia
convert(package::Module, geom) = convert(Val{nameof(package)}(), geom)
```

Reply:
Val actually works fine just with `Val{nameof(module)}()`. It even compiles away.

But I've changed it already to the second idea ;)